src/main/java/io/zipcoder/Classroom.java

```diff
@@ -1,69 +1,51 @@
 package io.zipcoder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.TreeMap;
+import java.util.Map;
+import java.util.Comparator;
 
-public class Classroom {
-    private Student[] students;
+
+public class Classroom{
+    private Student []students;
     // Helper fields
     private int studentsEnrolled;
     private int maxStudents;
     private int numOfExams = 3;
 
-    public Classroom(Student[] students, int studentsEnrolled, int maxStudents) {
+    public Classroom(Student students[], int studentsEnrolled, int maxStudents) {
         this.students = students;
         this.studentsEnrolled = studentsEnrolled;
         this.maxStudents = maxStudents;
     }
 
-    /**
-     * Empty constructor.  You get to decide what max students should default to.
-     */
-
-    /**
-     * Constructor where the caller defines wht maxStudents is.
-     * @param maxStudents
-     */
-    public Classroom(int maxStudents) {
-        for(int i = maxStudents; i<= maxStudents; i++){
-            Student student = new Student("Billy", "Bob", new int[3]);
-        }
-
-    }
-
-    /**
-     * Constructor that builds a class from an existing Student array.
-     * @param students
-     */
-    public Classroom(Student[] students) {
-
-    }
-
-    /**
-     * Adds a student to the class, and returns true.
-     * If you cannot add any more students, then return false and print an error statement.
-     * @param student
-     * @return
-     */
     public boolean addStudent(Student student) {
-        return false;
+        if(students.length >= 10){
+            return false;
+        }
+        else return true;
     }
 
-    /**
-     * Returns the student with the firstName and lastName, and removes them from the array.
-     * If there is no student, return null.
-     * NOTE: You're going to want to move some students around in the array so that there are no empty spaces in the
-     * middle.
-     * @param firstName
-     * @param lastName
-     * @return
-     */
-    public Student removeStudent(String firstName, String lastName) {
-        return null;
-    }
+    public Student[] removeStudent(String firstName, String lastName) {
+        Student student = new Student(firstName, lastName, new ArrayList<Double>(Arrays.asList(90.00)));
+            String nameToRemove = firstName + " " + lastName;
+            Student[] kid = new Student[students.length];
+            int seat = 0;
+            for (int i = 0; i < students.length; i++) {
+                if (students[i].equals(student)) {
+                    kid[seat] = students[i];
+                    seat++;
+                }
+            }
+            return kid;
```

Comment:
what? this will never work. students[i].equals(student) should never be true. and you don't remove anything.